website/src/client/layout.tsx

```diff
@@ -1,16 +1,17 @@
 import React, { useMemo } from "react"
 import { apolloClient } from "../apollo"
-import { UserProvider, useCredentials } from "../auth"
+import { useCredentials } from "../auth"
+import { Provider as GraphQLProvider } from "urql"
 
-const LayoutClient = (p: { children: any }) => {
-  const creds = useCredentials()
-  const token = creds?.signInUserSession.idToken.jwtToken ?? null
+export const LayoutClient = (p: { children: any }) => {
+  // const creds = useCredentials()
+  const token = useCredentials() ?? null; // uses credentials
```

Comment:
I would move this `?? null` to inside `useCredentials` so that callsites don't need it.

Reply:
Thanks, made the change